src/pages/documentation/adobe-campaign-standard/release-notes.md

```diff
@@ -12,47 +12,47 @@
 
 Major version update for [Adobe Campaign Standard workflows](./index.md) for Adobe Experience Platform Mobile SDKs on iOS compatible with Mobile Core 4.0.0. The current release includes the following changes:
 
-* Updated the minimum supported version to iOS 11.0.
-* Include XCFrameworks built with Xcode 14.1 with the GitHub release.
+- Updated the minimum supported version to iOS 11.0.
+- Include XCFrameworks built with Xcode 14.1 with the GitHub release.
```

Comment:
Can we use asterisk as a consistent list item delimiter? Current setting "consistent" is misleading and it takes just the nearby context when validating instead of the entire file. We can update this setting to "asterisk" under https://github.com/AdobeDocs/aep-mobile-sdkdocs/blob/main/.github/linters/.markdownlint.yml#L50-L51

Reply:
I looked at this file, and it's all `-`, so for _this_ one at least, we don't have to worry about it. I've updated the linter to use `*` though.

Reply:
@calebk1 can you also push the rule update to use asterisk so it applies for PRs from now on?